lib.nix

```diff
@@ -1,41 +1,28 @@
 let
   # Allow overriding pinned nixpkgs for debugging purposes via plutus_pkgs
-  fetchNixPkgs = let try = builtins.tryEval <plutus_pkgs>;
+  iohkNix = import (
+    let try = builtins.tryEval <iohk_nix>;
     in if try.success
-    then builtins.trace "using host <plutus_pkgs>" try.value
-    else import ./fetch-nixpkgs.nix;
+    then builtins.trace "using host <iohk_nix>" try.value
+    else
+      let
+        spec = builtins.fromJSON (builtins.readFile ./iohk-nix.json);
+      in builtins.fetchTarball {
+        url = "${spec.url}/archive/${spec.rev}.tar.gz";
+        inherit (spec) sha256;
+      });
+  fetchNixpkgs = iohkNix.fetchNixpkgs ./nixpkgs-src.json;
+  pkgs = import fetchNixpkgs { config = {}; overlays = []; };
```

Comment:
so above allows to override `iohk_nix` but not `nixpkgs` anymore?  (contrary to comment)

Reply:
Yeah, comment seems dead.

Reply:
Yeah, I'll fix the comment. Nixpkgs can be overrode using custom_pkgs now. I'll add a comment about that as well.